hosts/azure/builder/configuration.nix

```diff
@@ -46,7 +28,7 @@
   nix.settings.substituters = [
     # Configure Nix to use the bucket (through rclone-http) as a substitutor.
     # The public key is passed in externally.
-    "http://localhost:8080"
+    "localhost:8080"
```

Comment:
I think nix still wants the http:// scheme.

Reply:
oh yes that was a typo